test/schema.test.ts

```diff
@@ -2,12 +2,20 @@ const assert = require('assert'),
   zSchema = require('z-schema'),
   inspect = require('util').inspect;
 
-const schema = require('../lib/schema.json'),
-  schemautil = require('../src/schema/schemautil'),
-  specSchema = require('../src/schema/schema').schema;
+const schema = require('../lib/schema.json');
+
+import * as schemautil from '../src/schema/schemautil';
+import {schema as specSchema} from '../src/schema/schema';
 
 describe('Schema', function() {
   it('should be valid', function() {
+    zSchema.registerFormat('color', function (str) {
+      return false;
```

Comment:
Well, the formatter is not actually ever used because we only validate that the schema is valid.